visual-tests/scripts/visual-tests-build.mjs

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable no-console */
```

Comment:
`Can you describe the script's purpose and what it does? Just like we have in our other scripts, for example [release.mjs](https://github.com/handsontable/handsontable/blob/develop/scripts/release.mjs#L1-L6)`

Done.

`IMO the file name needs to be clarified. Right now, It says that something will be build, but in fact, it does more. It builds the examples, tests them, and saves the results as screenshots. What do you think to rename the file to just test.mjs?`

My idea was name it `build` cause it `builds` package of screenshots to `upload` in next file.

Reply:
What do you think @jansiegel, about the naming of that file? Because I'm still confused with the name `build`. If I had to choose I'd prefer `test.mjs` as the script not only builds but actually runs the tests using `playwright test` command.